core/embed/rust/src/ui/component/text/op.rs

```diff
@@ -97,14 +97,15 @@ impl<'a, T: StringType + Clone + 'a> OpTextLayout<T> {
                     };
                 }
                 // Drawing text
-                Op::Text(text) => {
+                Op::Text((text, is_complete)) => {
                     // Try to fit text on the current page and if they do not fit,
                     // return the appropriate OutOfBounds message
 
                     // Inserting the ellipsis at the very beginning of the text if needed
                     // (just once for the first Op::Text on the non-first page).
+                    // Also just for incomplete texts that were separated.
                     self.layout.continues_from_prev_page =
```

Comment:
we can directly set `self.layout.continues_from_prev_page = continued` (or `!is_complete` in the current version of this code), right? there is no way to get `Text(t, continued=true)` when we already processed some chars